### PR TITLE
Jira/tempo hide events

### PR DESCRIPTION
**Pull request by andrasguseo**

### CODE REVIEW

**Comment by andrasguseo** [comment]
Before:
https://dl.dropboxusercontent.com/s/7hglw7kcnwqyhcp/shot_200116_225604.jpg

After:
https://dl.dropboxusercontent.com/s/ic3aexhk3ahql8a/shot_200116_225758.jpg

You'll see that all "Nighttime" and "Busy with kids" entries are hidden.

**Comment by andrasguseo** [comment]
Thinking here...
The class which is checked on line 54 might be different for everyone...

File: jira/jira-mywork-hide-events.user.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    // Do we want error logging in the console?
+    var log = false;
+
+    // Wait 2 seconds...
+    var startScript = window.setTimeout( hideThem, 2000 );
+
+    function hideThem() {
+
+        // Collect all the events
+        slots =   document.getElementsByClassName( 'sc-fAjcbJ' );
```
Comment (sc0ttkclark) [inline jira/jira-mywork-hide-events.user.js]: This is the class on mine as well, FYI

Comment (andrasguseo) [reply jira/jira-mywork-hide-events.user.js]: Good to hear. So hopefully it'll work out of the box for others as well. 🤞

**Pull request merged**